src/utilities/Libify.js

```diff
@@ -459,6 +459,68 @@ Libify.Operation.revokeSponsorship = function(opts) {
   }
 }
 
+Libify.Operation.clawback = function(opts) {
+  assertNotEmpty(opts.asset, 'Clawback operation requires asset');
+  assertNotEmpty(opts.from, 'Clawback operation requires account from which the asset is clawed back');
+  assertNotEmpty(opts.amount, 'Clawback operation requires amount clawed back');
+  return Sdk.Operation.clawback({
+    asset: Libify.Asset(opts.asset),
+    from: opts.from,
+    amount: opts.amount,
+    source: opts.sourceAccount,
+  })
+}
+
+Libify.Operation.clawbackClaimableBalance = function(opts) {
+  assertNotEmpty(opts.balanceId, 'Clawback Claimable Balance operation requires claimable balance ID');
+
+  return Sdk.Operation.clawbackClaimableBalance({
+    balanceId: opts.balanceId,
+  });
+}
+
+Libify.Operation.setTrustLineFlags = function(opts) {
+  assertNotEmpty(opts.asset, 'Set Trust Line Flags operation requires asset');
+  assertNotEmpty(opts.trustor, 'Set Trust Line Flags operation requires trustor');
+
+  if (
+    !(opts.setFlags && opts.setFlags.length) &&
+    !(opts.clearFlags && opts.clearFlags.length)
+  ) {
+    throw new Error(
+      "Set Trust Line Flags operation requires at least one flag to be selected"
+    );
+  }
+
+  const flagLabels = {
+    authorized: "Authorized",
+    authorizedToMaintainLiabilities: "Authorized to maintain liabilites",
+    clawbackEnabled: "Clawback enabled",
+  };
```

Comment:
can we reuse `flagsClear` from above?

Reply:
Keeping this for now for consistency. https://github.com/stellar/laboratory/issues/523